pages/data/search.vue

```diff
@@ -384,15 +405,37 @@ export default {
       }
     },
     dataRecordHeaders() {
-      const headerKeys = [
-        'timestamp_date',
-        'content_category',
-        'platform_type',
-        'platform_id',
-        'instrument_name',
-        'processed_datetime',
-        'actions'
-      ]
+      let headerKeys = []
+      if (
+        this.selectedDataset === 'UV Index' ||
+        this.selectedDataset === 'Indice UV'
```

Comment:
This check can be swapped to use `this.selectedDatasetID === uv_index_hourly` so it's language agnostic.